### PR TITLE
calculate implemented

### PR DESCRIPTION
**Pull request by PawelYak**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @PawelYak

**Comment by ziobron** [comment]
⏱ 3 XP extra for @PawelYak for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+        if (second !=0){
+       return std::to_string(first/second);
+        }
+        return "Division by 0";
+   }
+   else if (command == "multiply"){
+       return std::to_string(first*second);
+   }
 }
+return "Invalid data";
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:24]: If you return variable here, you don't need to use `else if`

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+#include <cctype>
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+if (command == "divide" || "multiply" || "add" || "subtrac"){
+   
+   if(command == "add"){
+       return std::to_string(first+second);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:9]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+   }
+   else if (command == "subtract"){
+       return std::to_string(first-second);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:12]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/calculate/calculate.hpp [hunk 4]
```diff
@@ -1,7 +1,25 @@
...
+   }
+    else if (command == "divide"){
+        if (second !=0){
+       return std::to_string(first/second);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:16]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/calculate/calculate.hpp [hunk 5]
```diff
@@ -1,7 +1,25 @@
...
+        }
+        return "Division by 0";
+   }
+   else if (command == "multiply"){
+       return std::to_string(first*second);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:21]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/calculate/calculate.hpp [hunk 6]
```diff
@@ -1,7 +1,25 @@
...
+   if(command == "add"){
+       return std::to_string(first+second);
+   }
+   else if (command == "subtract"){
+       return std::to_string(first-second);
+   }
+    else if (command == "divide"){
+        if (second !=0){
+       return std::to_string(first/second);
+        }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:17]: Wrong indentation

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**